### ISSUE TITLE
The website language should be properly reflected in the meta lang tag

### ISSUE THREAD

**Issue by phacks** [theodo/falco#141]
Thanks @AlbericTrancart for the report!

From https://a11yproject.com/checklist/#section-global-code:

> This helps screen readers to pronounce content correctly.

Today the `meta lang="en"` is hardcoded and does not reflect the language of the current user.
